data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,127 @@ mission "Strider notification"
 			scene `thumbnail/hai pond strider`
 			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoil the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
 				decline
+
+
+
+mission "Hai Film Crew [1]"
+	name "Hai Film Crew"
+	description "Bring a movie crew from <origin> to <destination>, so that they can film a horror movie there."
+	minor
+	source
+		attributes hai urban
+		not attributes "uninhabited" "station"
+	destination "Darkrest"
+	passengers 80
+	cargo "film equipment" 50
+	to offer
+		random < 30
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`A rapid tapping on your shoulder dissappiates any intentions you might have had about touring this spaceport in peace. You turn around to find a Hai, short even by their standards, looking up at you.`
+			`	"You're a ship captain, yes? Would you like to be part of cinema history?"`
+			choice
+				`	"I'm not much of a movie buff. I'll pass."`
+					decline
+				`	"Sure, what do you need me to do?"`
+				`	"What do you mean?"`
+			`	"My name is Likeet, and I'm in charge of coordinating transportation for an independent film crew. We need you to bring <bunks> crew members and <tons> of film equipment to Darkrest, then bring us back after about a month. We can pay you <payment>, but the rewards of being involved in an ambitious project such as this will be priceless."`
+			choice
+				`	"Why would you want to go to <planet>? Isn't that an abandoned station?"`
+					goto ask
+				`	"Sure, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label ask
+			`	"Ah, you must not know about <planet>. There are many tales which swirl around the station, chief among them being the existence of ghosts and monsters haunting its corridors. We'd like to take advantage of the popular image of Darkrest in the creation of our horror film by using it as a backdrop. It's sort of the human equivalent of setting your film in a 'haunted' forest. Don't worry, we'll bring enough food to last us for the month."`
```

Comment:
What if the player has heard the ghost stories? I'm not sure if that's currently possible gameplay-wise, but it doesn't sound unlikely lore-wise.

Reply:
There currently isn't any mention of the ghost stories anywhere in the game, that was introduced with this PR and #9601. Neither are in the game yet.